dev/devicelab/lib/tasks/hot_mode_tests.dart

```diff
@@ -48,9 +48,12 @@ TaskFunction createHotModeTest() {
           process.stdout
               .transform<String>(utf8.decoder)
               .transform<String>(const LineSplitter())
-              .listen((String line) {
+              .listen((String line) async {
             if (line.contains('\] Reloaded ')) {
               if (hotReloadCount == 0) {
+                // Ensure that notifications triggered after the previous hot
+                // reload have finished.
+                await Future<void>.delayed(const Duration(seconds: 2));
```

Comment:
Could we instead wait for a 'Done' message?

Reply:
We're getting the hot reload done message at the correct time, but because these devices are so completely hosed, we're getting a bunch of VM notifications from the hot restart several seconds after it has completed.

Reply:
Could we somehow ask the VM to flush any pending notifications related to hot reload instead?  I'm worried about this flaking (understanding that it's meant to fix a flake)

Reply:
That capability does not exist as far as I know

Reply:
Better idea, just wait for a few seconds after starting the app

Reply:
Don't we have a pattern for that elsewhere - listen to stdout for a message from the app that it's started?

Reply:
run_hot has its own benchmark mode logic, I think it should be implemented there.